projects/1.21-fabric/assets/simple-shulker-preview/simpleshulkerpreview/lang/zh_cn.json

```
@@ -0,0 +1,93 @@
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsGeneral.translateY": "垂直偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsGeneral.translateY.@Tooltip": "图标的垂直偏移量，从上到下。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsGeneral.translateZ": "叠加层偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsGeneral.translateZ.@Tooltip": "图标的叠加层偏移量。当其他模组覆盖/被此模组覆盖时使用。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsGeneral.scale": "图标大小",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsGeneral.scale.@Tooltip": "图标的大小。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle": "图标位置 - 收纳袋",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.@Tooltip": "收纳袋上的图标位置。默认不同于潜影盒的图标位置，以避免与原版的容量指示器重叠。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.translateX": "水平偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.translateX.@Tooltip": "图标的水平偏移量，从左到右。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.translateY": "垂直偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.translateY.@Tooltip": "图标的垂直偏移量，从上到下。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.translateZ": "叠加层偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.translateZ.@Tooltip": "图标的叠加层偏移量。当其他模组覆盖/被此模组覆盖时使用。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.scale": "图标大小",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsBundle.scale.@Tooltip": "图标的大小。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked": "图标位置 - 堆叠潜影盒",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.@Tooltip": "堆叠潜影盒上的图标位置。默认不同于潜影盒的图标位置，以避免与数量指示器重叠。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.translateX": "水平偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.translateX.@Tooltip": "图标的水平偏移量，从左到右。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.translateY": "垂直偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.translateY.@Tooltip": "图标的垂直偏移量，从上到下。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.translateZ": "叠加层偏移量",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.translateZ.@Tooltip": "图标的叠加层偏移量。当其他模组覆盖/被此模组覆盖时使用。",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.scale": "图标大小",
+	"text.autoconfig.simpleshulkerpreview.option.iconPositionOptionsStacked.scale.@Tooltip": "图标的大小。",
+	"text.autoconfig.simpleshulkerpreview.option.shulkerInventoryOptions": "潜影盒界面选项",
+	"text.autoconfig.simpleshulkerpreview.option.shulkerInventoryOptions.@Tooltip": "当服务端模组需要修改客户端潜影盒属性时的选项。",
```

```suggestion
	"text.autoconfig.simpleshulkerpreview.option.shulkerInventoryOptions.@Tooltip": "用以按服务端模组的有关修改调节客户端潜影盒属性的选项。",
```
Options to modify the client-side shulker attributes when server-side mods *(mods' modifications)* require it.（括号是推测的）
*或需上机验证？*